documentation/feature/medical-system.md

```diff
@@ -125,83 +144,101 @@ Advanced medical system brings more different types of wounds, each with its own
-  * Velocity wounds - high effectiveness
-  * Puncture wounds - high effectiveness
+* QuikClot is best for: 
+  * All types of wounds - medium effectiveness
+  * Special: It has lowest wound opening chance. It is often needed to apply it more than once, depening on wound type and size.
 
 * Tourniquet
   * Can only be applied on limbs
   * Stops bleeding from wounds
+  * Should be used only when there are more than one wounds
   * Should be taken off as fast as possible and applied only to give medic time to bandage all the wounds
   * If not taken off for a while it will cause pain to patient, can cause death that way
 
 #### 3.1.6 Transfuzions
-In case of blood loss, blood could be replenished by three different types of IV: blood, plasma and saline. We distinguish 3 different sizes of IV bags: 250ml, 500ml and 1000ml.
+
+In case of blood loss, identified by lower blood pressure, blood could be replenished by three different types of IV: blood, plasma and saline. We distinguish 3 different sizes of IV bags: 250ml, 500ml and 1000ml.
 Transfuzing 250ml of given IV takes about 1 minute to complete!
 
-+#### 3.1.7 PAK and Surgical kit
-Using Personal Aid Kit brings patient to the best possible health state. Depending on module options, PAK can be used only in specified situations, specified place and by specified person.
+#### 3.1.7 PAK and Surgical kit
+
+Using Personal Aid Kit brings patient to the best possible health state. Depending on module options, PAK can be used only in specified situations, such as when all wounds are bandaged and patient is awake, specified place, such as medical vehicle or building, and by specified person, such as medic or doctor.
+
 Surgical kit is used to stich bandaged wounds so they will never open again. Depending on module options, surgical kit can be used only in specified situations, specified place and by specified person.
 
-### 3.2 Advanced medical system - recommended gear
- 
+### 3.2 Healing guideline
+
+Below you will find how to handle wounds if you play as a soldier or a medic.
+
+#### 3.2.1 Healing yourself as a soldier
+
+Follow steps below.
+
+ 1. Are you bleeding? If yes go to step 2, if no, go to step 5.
+ 2. Did your limbs got hit? If yes, apply tourniquet on them.
+ 3. Start applying proper bandages on your wounds.
+ 4. Remember to take off your tourniquet!
+ 5. Do you feel pain? If no, skip step below.
+ 6. Check your heart rate, if it's normal, apply morphine. You should be fine now.
+ 7. Always call for a medic if your wounds were severe. Tell him what wounds you sustained and what medications you used.
+
+#### 3.2.2 Healing other soldiers as a medic
+
+Follow steps below.
+
+ 1. Most important is to check if patient is conscious and how many wounds he sustained. Quickly check pulse and blood pressure. If there is no pulse, go to next step, if there is pulse but patient is unconscious, then you should do everything to stabilize patient status and he will wake up on his own after a while.
```

Comment:
This is not according to the TCCC I believe. while you always first check some vitals, the next step is always to stop any kind of external bleeding (or free airway, but we don't have that yet). 

Has this been discussed in the medical channel on slack?